app/Actions/DisplayCurrentConfiguration.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Actions;
+
+use function is_bool;
+use function is_string;
+use App\Support\BaseAction;
+
+class DisplayCurrentConfiguration extends BaseAction
+{
+    /**
+     * Displays current configuration.
+     *
+     * @return void
+     */
+    public function __invoke(): void
+    {
+        $rows = config('lambo', []);
+
+        $rows = collect($rows)->filter(function ($item, $key) {
+            return $key !== 'after';
+        })->map(function ($item, $key) {
+            if (is_bool($item)) {
```

Comment:
can we pull this translation to a private method to make it clearer? just everything in this map except the return at the end